src/views/PageSearchPortal.vue

```diff
@@ -1,6 +1,11 @@
 <template>
     <b-tabs v-model="activeTabView">
         <b-tab-item label="Parking Request (VO/SO)">
+            <div class="request-search-control">
+                <p></p>
```

Comment:
why this empty <p>
if its for a new line, then please use <br />

Reply:
<p>  was removed